torch/nn/modules/loss.py

```diff
@@ -1029,14 +1029,17 @@ def forward(self, input: Tensor, target: Tensor) -> Tensor:
 
 
 class CrossEntropyLoss(_WeightedLoss):
-    r"""This criterion computes the cross entropy loss between input and target.
+    r"""This criterion computes the cross entropy loss between input logits
+    and target.
 
     It is useful when training a classification problem with `C` classes.
     If provided, the optional argument :attr:`weight` should be a 1D `Tensor`
     assigning weight to each of the classes.
     This is particularly useful when you have an unbalanced training set.
 
-    The `input` is expected to contain raw, unnormalized scores for each class.
+    The `input` is expected to contain the logits for each class (which do `not` need
+    to be positive or sum to 1, in general). This class will compute a softmax
```

Comment:
note that a LogSoftmax is actually computed, not a softmax. I suggest leaving out this sentence

Reply:
Removed